jerry-main/main-unix.c

```diff
@@ -107,6 +107,21 @@ assert_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< f
   }
 } /* assert_handler */
 
+/**
+ * Provide the '__gc' implementation for the engine.
```

Comment:
To me, (double) underscore prefix is arbitrary as I haven't seen it in use anywhere else in the code base (not referring to `assert` or `print` only). I'll vote for `gc()`, but let others speak their mind, too.